app/views/facility_orders/_merge_order_form.html.haml

```diff
@@ -2,7 +2,7 @@
   .well
     = simple_form_for [current_facility, @add_to_order_form], url: facility_order_path(current_facility, @order), method: :put, html: { class: "js--edit-order" } do |f|
       - if SettingsHelper.feature_on?(:cross_core_projects) && can?(:available_for_cross_core_ordering, Product)
-        - facilities = @add_to_order_form.facilities.map { |f| [f.name, f.id, {"data-products-path" => available_for_cross_core_ordering_facility_products_path(f, format: :js)}] }
+        - facilities = @add_to_order_form.facilities.map { |f| [f.name, f.id, {"data-products-path" => available_for_cross_core_ordering_facility_products_path(f, format: :js), "data-original-order-facility" => @order.facility_id}] }
```

Comment:
What do you think about moving this into a method like `facilities_options` on the form object?  Then in the view we can have something like:
```
        = f.input_field :facility_id,
          collection: @add_to_order_form.facilities_options,
          class: "js--edit-order__facility js--chosen",
          label: Facility.model_name.human,
          include_blank: false
```